### DISCUSSION TITLE
Help supporting more developers on MagicBell?

### DISCUSSION THREAD

**Discussion by JoeyDeBruin** [magicbell/community#163] [General]
Hey there, we run [big programs](https://backdropbuild.com/) with about 2000 developers where we support builders in launching something new. We also work with companies (totally free of charge) like Stability, AWS, Convex, and others to encourage and support builders on their tech. We've seen some interest in MagicBell from builders in past programs and I was hoping to chat with someone on the team about making it easier to build on MagicBell in the upcoming program? Thanks!

**Comment by markotoole1** [comment]
Hi @JoeyDeBruin, thank you for the message. I would be happy to have a call with you on what you might have in mind. Please email me at mark@magicbell.io and we can set up some time.
